packages/client-search/src/types/SearchOptions.ts

```diff
@@ -334,4 +334,12 @@ export type SearchOptions = {
    * Engine's default: true
    */
   readonly enableABTest?: boolean;
+
+  /**
+   * Enable word segmentation (also called decompounding) at query time for
+   * languages that have it enabled using "decompoundedAttributes". For example,
```

Comment:
```suggestion
   * compatible languages (Danish, Dutch, Finnish, German, Norwegian, Swedish) . For example,
```

Reply:
This setting is not linked to decompoundedAttributes, beyond its behaviour. You can use one or the other only independantly (that would be weird, but possible).

Reply:
is this list never going to change? I'd rather have the website be the source of truth for the languages list

Reply:
It might, but not in the foreseeable future. 
That being said you make a good point, so I'll leave it up to you to decide. :)

Reply:
I removed the mention of decompoundedAttributes for now, let me know what you think